### PR TITLE
Article about notable changes for developers in v8

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | This article summarizes actions that are required from developers to migrate their solutions from 1.7 to v8
| Fixed ticket? | n/a

### CODE REVIEW

File: news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+date: 2022-08-12
+authors: [kpodemski, PabloBorowicz]
+icon: icon-road
+tags:
+- tutorial
+- major
+- 8.0
+---
+
+PrestaShop 8 beta version was released this week. If you are a developer in our ecosystem, create modules or themes, and have clients working with PrestaShop, this is an excellent opportunity to spend some time with this version and test your solutions.
```
Comment (PrestaEdit) [inline news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md]: You could add the link to the specific blog post, maybe.

File: news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+In this article, I will focus on things that might affect your work. You will read here about changes that are not backward compatible, and could require you to make changes in the code of your solutions.
+
+It is worth noting that PrestaShop 8 is a new major version. Although it is an evolution rather than a revolution, you will find several software code changes that can significantly impact some of you.
+
+As always, we have done our best to make all important information about changes available in the [developer documentation](https://devdocs.prestashop.com). This article is an introduction and explanation to the content you will find there.
```
Comment (PrestaEdit) [inline news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md]:
```suggestion
As always, we have done our best to make all important information about changes available in the [developer documentation](https://devdocs.prestashop.com/8/). This article is an introduction and explanation to the content you will find there.
```

**Review by coffrep — APPROVED** [review]

**Comment by coffrep** [comment]
Good for me, I've approved.

File: news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md (hunk 1 above)
Comment (kpodemski) [inline news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md]:
```suggestion
PrestaShop 8 beta version was [released last week.](https://build.prestashop.com/news/prestashop-8-0-beta-release/) If you are a developer in our ecosystem, create modules or themes, and have clients working with PrestaShop, this is an excellent opportunity to spend some time with this version and test your solutions.
```

File: news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "Migrate your solutions from PrestaShop 1.7 to 8.0"
+subtitle: "Summary of notable changes for developers"
+date: 2022-08-12
```
Comment (kpodemski) [inline news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md]:
```suggestion
date: 2022-08-15
```

File: news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+authors: [kpodemski, PabloBorowicz]
```
Comment (kpodemski) [inline news/_posts/2022-08-12-migrate-your-solutions-from-17-to-80.md]:
```suggestion
authors: [kpodemski]
```

**Pull request merged**